platform.md

```diff
@@ -546,6 +546,7 @@ Usage:
 | `<stack>`           | `CNB_STACK_PATH`           | `/cnb/stack.toml`   | Path to stack file (see [`stack.toml`](#stacktoml-toml)
 | `<uid>`             | `CNB_USER_ID`              |                     | UID of the build image `User`
 | `<layers>/config/metadata.toml` | | | Build metadata (see [`metadata.toml`](#metadatatoml-toml)
+|                     | `SOURCE_DATE_EPOCH`        |                     | Timestamp for `created` time in app image config                                           |
```

Comment:
Though other variables used to configure the lifecycle start with `CNB_` it seems better keep the interface consistent with other tooling (see under "Reading the variable" [here](https://reproducible-builds.org/docs/source-date-epoch/)).